bindings/go/rsa/signing/handler/handler.go

```diff
@@ -0,0 +1,292 @@
+// Package handler implements RSA signing and verification for OCM.
+// It supports both RSASSA-PSS and RSASSA-PKCS1-v1_5, and two encodings:
+//  1. Plain: hex signature bytes without certificates.
```

Comment:
Nit: technically, the private key can use a certificate it's just not embedded in the output. Right?